templates/error.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <title>Build error</title>
+    <script src="/ember-cli-live-reload.js" type="text/javascript"></script>
```

Comment:
I believe the perfect solution, that would prevent polluting broccoli core, would be:

* ability to swap out `dir` + `error` templates
* ability to register handlebars helpers

Quirks of babel errors, they need to be cleaned-up for our use-case, as they include the nice error message in `error.message` and `error.stack`. They also always include the ansi-color sequences if the terminal supports them. Which is nice for the errors we log to the console, but not so nice for the web-browser.